src/ipc-handlers.ts

```diff
@@ -596,27 +596,33 @@ export function openTerminalAtPath( _event: IpcMainInvokeEvent, targetPath: stri
 		const platform = process.platform;
 		const cliPath = nodePath.join( getResourcesPath(), 'bin' );
 
+		const exePath = app.getPath( 'exe' );
+		const appDirectory = app.getAppPath();
+		const appPath =
+			process.env.NODE_ENV === 'development' ? `${ exePath } ${ appDirectory }` : exePath;
+
 		let command: string;
 		if ( platform === 'win32' ) {
 			// Windows
-			command = `start cmd /K "set PATH=${ cliPath };%PATH% && set APP_PATH=${ app.getPath(
-				'exe'
-			) } && cd /d ${ targetPath }"`;
+			command = `start cmd /K "set PATH=${ cliPath };%PATH% && set APP_PATH=${ exePath } && cd /d ${ targetPath }"`;
 		} else if ( platform === 'darwin' ) {
 			// macOS
 			const script = `
-			tell application "Terminal"
-				if not application "Terminal" is running then launch
-				do script "clear && export PATH=${ cliPath }:$PATH && export APP_PATH=${ app.getPath(
-					'exe'
-				) } && cd ${ targetPath }"
-				activate
-			end tell
-			`;
+	tell application "Terminal"
+		if not application "Terminal" is running then launch
+		do script "clear && export PATH=${ cliPath }:$PATH && export APP_PATH=${ exePath } && cd ${ targetPath }"
+		activate
+	end tell
+	`;
 			command = `osascript -e '${ script }'`;
 		} else if ( platform === 'linux' ) {
 			// Linux
-			command = `gnome-terminal --working-directory=${ targetPath }`;
+			command = `export PATH=${ cliPath }:$PATH && export APP_PATH="${ appPath }"`;
+			if ( process.env.NODE_ENV === 'development' ) {
+				command += ` && export EXE_PATH="${ exePath }" && export APP_DIRECTORY="${ appDirectory }"`;
+			}
```

Comment:
I presume this enables running `wp-cli` commands when building the app using `npm start`, is this accurate?

Reply:
Exactly! I didn't found a better way to do it so, as concat those two, and pass it as one parameter, throw an error.

Reply:
Great. We could probably apply the same approach to macOS and Windows on another PR.